src/components/general/DatePicker.js

```diff
@@ -126,14 +104,13 @@ class DatePicker extends Component {
 			day,
 			year,
 			monthArr,
-			dayArr,
 			yearArr,
 			focused
 		} = this.state;
 
 		let iconSize = 32;
 
-		if (!focused)
+		if (!focused) {
 			return (
 				<View>
```

Comment:
I know you didn't change this but i'm 60% sure you don't need a View here.

Reply:
You don't